### PR TITLE
add: basic react and testing setup

### PR DESCRIPTION
**Pull request by MehulKChaudhari**
PR will add basic setup of UI and testing from the mono repo created earlier for feature-flag service [link](https://github.com/Real-Dev-Squad/feature-flag-service)

### CODE REVIEW

File: src/Components/Login/Login.js
Comment (prakashchoudhary07) [inline src/Components/Login/Login.js:1]: Empty files?

File: src/Components/Logo/Logo.js
Comment (prakashchoudhary07) [inline src/Components/Logo/Logo.js:1]: Re?

File: src/Components/Search/Search.js
Comment (prakashchoudhary07) [inline src/Components/Search/Search.js:1]: Re?

File: package.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    "react-dom": "^18.1.0",
+    "react-scripts": "5.0.1",
+    "web-vitals": "^2.1.4"
+  },
+  "scripts": {
+    "start": "react-scripts start --mode=develoment",
+    "build": "react-scripts build --mode=production",
+    "test-watch": "react-scripts test --coverage",
+    "test": "react-scripts test --watchAll=false",
+    "eject": "react-scripts eject",
```
Comment (prakashchoudhary07) [inline package.json:22]: What does this do?

**Comment by MehulKChaudhari** [comment]
Closing this PR, because we want to use Vite and Vitest so the PR #2

**Pull request closed without merging**